plugins/netskope/plugin.spec.yaml

```diff
@@ -0,0 +1,413 @@
+plugin_spec_version: v2
+extension: plugin
+products: [insightconnect]
+name: netskope
+title: Netskope
+description: Connected with Netskope, this plugin allows to update a file hash list and use it as an allowlist or blocklist for files in your organization, as well as managing URL lists by creating a new URL list or editing existing one. In addition, Netskope plugin has an action that allows to check single user's confidence index
```

Comment:
I would try and write a more concise description here.
Try and avoid using `.` throughout the description. Using `,` is okay however.
https://docs.rapid7.com/insightconnect/style-guide/#punctuation

Reply:
You are right. We will change that.